contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -6081,13 +6081,18 @@ exec_stmt_close(PLtsql_execstate *estate, PLtsql_stmt_close *stmt)
 static int
 exec_stmt_commit(PLtsql_execstate *estate, PLtsql_stmt_commit *stmt)
```

Comment:
Do we need any changes in `exec_stmt_commit`?

Reply:
No will revert